device/core/devdoc/device_client_requirements.md

```diff
@@ -229,6 +229,16 @@ interface DeviceMethodEventHandler {
 
 **SRS_NODE_DEVICE_CLIENT_13_021: [** `onDeviceMethod` shall throw a `NotImplementedErrorError` if the underlying transport does not support device methods. **]**
 
+#### setRetryPolicy(policy)
+
+**SRS_NODE_DEVICE_CLIENT_16_083: [** The `setRetryPolicy` method shall throw a `ReferenceError` if the policy object is falsy. **]**
+
+**SRS_NODE_DEVICE_CLIENT_16_084: [** The `setRetryPolicy` method shall throw an `ArgumentError` if the policy object doesn't have a `shouldRetry` method. **]**
+
+**SRS_NODE_DEVICE_CLIENT_16_085: [** The `setRetryPolicy` method shall throw an `ArgumentError` if the policy object doesn't have a `nextRetryTimeout` method. **]**
+
+**SRS_NODE_DEVICE_CLIENT_16_086: [** Any operation happening after a `setRetryPolicy` call should use the policy set during that call. **]**
```

Comment:
>operation [](start = 41, length = 9)

This is crazy vague.  What's an operation anyway?

Reply:
added examples. not sure how to make that very precise.

---
In reply to: [145962392](https://github.com/Azure/azure-iot-sdk-node/pull/114#discussion_r145962392) [](ancestors = 145962392)